### ISSUE TITLE
iatibugtracker.org seems broken :(

### ISSUE THREAD

**Issue by andylolz** [VincentVW/iati_bugs#10]
Site appears to be down… Is it possible to give it a kick?

**Comment by VincentVW** [comment]
Hi Andy! I don't own the domain :) 

@siemvaessen can you give it a kick?

**Comment by VincentVW** [comment]
(Or ship me the domain and I'll make sure it runs somewhere, I'm fine with either, don't want to cause work for anyone else here)

**Comment by siemvaessen** [comment]
Yeah, killed it and missed this thread! Undocumented and unsupported stuff is off our table. @VincentVW if you could document it (install, reqs etc) and it makes use of the Python 3 version of OIPA I'd be happy to reconsider.

**Comment by siemvaessen** [comment]
@VincentVW if you tell me where you want to run it, I'll DNS-it!

**Comment by siemvaessen** [comment]
@VincentVW bump up!
